docs/scaladoc-gen.md

```diff
@@ -0,0 +1,39 @@
+ScaladocGenPlugin
+=================
+
+**Boss**: Jesse
+
+**Wraps**: [`sbt-unidoc`](https://github.com/sbt/sbt-unidoc), [`sbt-ghpages`](https://github.com/sbt/sbt-ghpages), and [`sbt-site`](https://github.com/sbt/sbt-site).
+
+A plugin to generate Scaladoc for aggregate (root) projects in sbt.
+
+See the sites plugin for more instructions on how to configure it, if you're interested in having more things on your github site than just scaladoc!
+
+Usage
+=====
+Configuring:
+```scala
+// Enable the plugin on your project, which should aggregate all of your
+// subprojects.
+lazy val myRootProject = project.in(file(".")).aggregate(one, two)
+  .settings(
+    // This setting is required in order for the plugin to load.
+    scaladocGenGitRemoteRepo := "git@github.com:allenai/myprojectname.git",
+    // This is not strictly required, but is needed if you want your library to
+    // be linkable by other projects in their scaladoc.
+    apiURL := Some(url("https://allenai.github.io/myprojectname/latest/api/"))
```

Comment:
According to the [sbt docs](http://www.scala-sbt.org/0.13/docs/Howto-Scaladoc.html), this embeds something in the POM that clients can read when generating docs, so that you don't need to manually configure API mappings.